### PR TITLE
Release version 1.0.0

### PR DESCRIPTION
**Pull request by hakontro**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+
+## Breaking changes
+* All schemas updated to match API v1
+
+## Enhancements
+* Parallel retrieval is now a lot faster, and parallelity can be specified using the `partitions` option.`
+
+* All datetime columns are now Timestamps rather than milliseconds since Epoch.
+
+* Format has been shortened, for convenience: `.format(cognite.spark)`.
```
Comment (sebbert) [inline CHANGELOG.md]: Should be `.format("cognite.spark")`

Comment (hakontro) [reply CHANGELOG.md]: Absolutely, thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+# 1.0.0
+This release goes from using Cognite API version 0.5/0.6 to using [Cognite API v1](https://docs.cognite.com/api/v1/). 
+All reads from, and writes to, CDF now uses the [Cognite Scala SDK](https://github.com/cognitedata/cognite-sdk-scala).
```
Comment (sebbert) [inline CHANGELOG.md]: it uses, they use

**Comment by sebbert** [comment]
Apologies if I'm being too pedantic about grammar here

File: CHANGELOG.md (hunk 2 above)
Comment (hakontro) [reply CHANGELOG.md]: True! :man_facepalming: Good catch, please keep being pedantic about grammar.

Comment (sebbert) [reply CHANGELOG.md]: I think that goes without saying 😉

File: CHANGELOG.md (hunk 1 above)
Comment (wjoel) [reply CHANGELOG.md]: We should make this `cognite.spark.v1`

**Pull request merged**